Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -107,9 +107,6 @@ void BlockFormattingContext::compute_width(Box const& box, AvailableSpace const&
 
     auto const& computed_values = box.computed_values();
 
-    if (should_treat_width_as_auto(box, available_space) && available_space.width.is_intrinsic_sizing_constraint())
-        return;
```

Comment:
Unfortunately it's not this simple. I'm not exactly sure how this needs to work, but here's an example of something that will break with this change:

```html
<!DOCTYPE html><html><head><style>
  .outer {
    display: inline-block;
  }
  .inner {
    background: orange;
    min-width: 32px;
  }
</style></head><body><div class="outer"><div class="inner">well hello friends</div></div>
```